test/tests/30-audio

```diff
@@ -11,6 +11,23 @@ if [ -z "$release" ]; then
 fi
 
 snapshot "$release" core
+
+# FIXME: We are lacking a test for arch non-multilib here...
```

Comment:
Added a simple test (no pulseaudio): it's mostly the crouton installer that I want to test.